### DISCUSSION TITLE
Any way to cap the popup max size?

### DISCUSSION THREAD

**Discussion by TsukiZero** [uBlockOrigin/uBlock-issues#2247] [Q&A]
![image](https://user-images.githubusercontent.com/78157224/188785051-f0a92cd1-4c53-4b69-b448-a12c7e902b0e.png)
Because certain hostnames can be quite mean.

**Comment by ghost** [comment]
I see no issue:

![image](https://user-images.githubusercontent.com/886325/188876299-e6fb5a5e-8c20-4074-8cb3-c252770f341f.png)

**Comment by TsukiZero** [comment]
![image](https://user-images.githubusercontent.com/78157224/188904591-ef69bc55-4645-4794-a078-b6909565f467.png)
if you look at the way the hostname deforms the box and increases the button width it should be painfully obvious. (normal width for reference)

**Reply by ghost** [reply]
Works as designed. Extends to certain point and then folds the hostname.

**Reply by TsukiZero** [reply]
I'd rather have a way to set it so it doesn't show the hostname in full and to preserve the main pane size. like it could show part of the hostname, and preferably a clickable ". . ." that would toggle the full hostname until the popup is closed.

**Reply by uBlock-user** [reply]
> so it doesn't show the hostname in full and to preserve the main pane size. like it could show part of the hostname

That used to be old behaviour.

**Reply by TsukiZero** [reply]
> > so it doesn't show the hostname in full and to preserve the main pane size. like it could show part of the hostname
> 
> That used to be old behaviour.

And IMO it should have been preserved in some manner, or given a way to style it back in without requiring UserChrome.
